fetcher/primaryip.go

```diff
@@ -24,7 +24,7 @@ func (primaryIP primaryIP) Run(client *hcloud.Client) error {
 	for _, p := range primaryIPs {
 		datacenter := p.Datacenter
 
-		hourlyPrice, monthlyPrice, err := primaryIP.pricing.PrimaryIP(p.Type, datacenter.Name)
+		hourlyPrice, monthlyPrice, err := primaryIP.pricing.PrimaryIP(p.Type, datacenter.Location.Name)
```

Comment:
This is the correct change, I guess. I was wondering anyway, why Primary IP prices are based on datacenter and not on location, like everything else. Could you also adopt the `NewPrimaryIP` constructor to reflect that? Currently the Prometheus label is still called "datacenter", which should now be "location", like the others.